packages/playground/src/explorer/components/node_details_cards/farm_details_card.vue

```diff
@@ -39,12 +43,22 @@ export default {
     };
 
     const getFarmDetails = (): NodeDetailsCard[] => {
+      let farmId, farmName, farmStellarAddress;
+      if (props.node) {
+        farmId = props.node.farmId.toString();
+        farmName = props.node.farm.name;
+        farmStellarAddress = props.node.farm.stellarAddress;
```

Comment:
I don't think this is needed.
```suggestion
```

Reply:
This is needed for handling the card contents based on which page it is being used in (i.e nodes page or farms page)